Dockerfile

```diff
@@ -8,7 +8,7 @@
 # runtime performance.
 
 ARG RUST_IMAGE=rust:1.32.0
-ARG RUNTIME_IMAGE=gcr.io/linkerd-io/base:2017-10-30.01
+ARG RUNTIME_IMAGE=gcr.io/linkerd-io/proxy:git-820f9cd8
```

Comment:
Note that this dockerfile now uses a proxy image so that we can just overwrite the binary. This is now necessary because we require utilities built in the other repo.